services/appnavstate/test/src/main/kotlin/io/element/android/services/appnavstate/test/FakeAppForegroundStateService.kt

```diff
@@ -9,19 +9,29 @@ package io.element.android.services.appnavstate.test
 
 import io.element.android.services.appnavstate.api.AppForegroundStateService
 import kotlinx.coroutines.flow.MutableStateFlow
-import kotlinx.coroutines.flow.StateFlow
 
 class FakeAppForegroundStateService(
-    initialValue: Boolean = true,
+    initialForegroundValue: Boolean = true,
+    initialIsInCallValue: Boolean = false,
+    initialIsSyncingNotificationEventValue: Boolean = false
 ) : AppForegroundStateService {
-    private val state = MutableStateFlow(initialValue)
-    override val isInForeground: StateFlow<Boolean> = state
+    override val isInForeground = MutableStateFlow(initialForegroundValue)
+    override val isInCall = MutableStateFlow(initialIsInCallValue)
+    override val isSyncingNotificationEvent = MutableStateFlow(initialIsSyncingNotificationEventValue)
 
-    override fun start() {
+    override fun startObservingForeground() {
         // No-op
     }
 
     fun givenIsInForeground(isInForeground: Boolean) {
-        state.value = isInForeground
+        this.isInForeground.value = isInForeground
+    }
+
+    override fun updateIsInCallState(isInCall: Boolean) {
```

Comment:
Only used in test, can you double check please?

Reply:
It's not though?

<img width="1001" alt="image" src="https://github.com/user-attachments/assets/418e9e07-6c81-454b-9513-14628497e816" />

You're looking at the fake implementation, so it's normal that one will only be used in tests.